app/assets/javascripts/games.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```

Comment:
You can remove these blank files. Since rails does a lot of autoloading of files, blank files like this can start to drag down asset compilation since they still need to get hashed.